sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -318,6 +319,13 @@ def GetHeaders(  # pylint: disable=too-many-statements,too-many-branches
     if options.get("correlatedActivityId"):
         headers[http_constants.HttpHeaders.CorrelatedActivityId] = options["correlatedActivityId"]
 
+    if resource_type is "docs" and verb is not "get":
+        contentResponseOnWriteDisabled = options.pop(
```

Comment:
I thought for query, the verb will be post as well, will the header impact the response?